tests/core/pyspec/eth2spec/test/altair/block_processing/test_process_sync_aggregate.py

```diff
@@ -113,6 +113,36 @@ def test_invalid_signature_missing_participant(spec, state):
     yield from run_sync_committee_processing(spec, state, block, expect_exception=True)
 
 
+@with_altair_and_later
+@spec_state_test
+@always_bls
+def test_invalid_signature_no_participants(spec, state):
+    committee_indices = compute_committee_indices(spec, state, state.current_sync_committee)
+
+    block = build_empty_block_for_next_slot(spec, state)
+    # Exclude one participant whose signature was included.
+    block.body.sync_aggregate = spec.SyncAggregate(
+        sync_committee_bits=[False for _ in committee_indices],
+        sync_committee_signature=b'\x00' * 96
+    )
+    yield from run_sync_committee_processing(spec, state, block, expect_exception=True)
+
+
+@with_altair_and_later
+@spec_state_test
+@always_bls
+def test_invalid_signature_infinite_signature_with_participants(spec, state):
```

Comment:
probably also want a test with infinite signature, 0 participants, which should succeed.

Reply:
Nvm, the success case is tested with `test_sync_committee_rewards_empty_participants` already